server/service/client_mdm.go

```diff
@@ -372,3 +372,17 @@ func (c *Client) prepareAppleMDMCommand(rawCmd []byte) ([]byte, error) {
 	}
 	return b, nil
 }
+
+func (c *Client) MDMLockHost(hostID uint) error {
+	if err := c.authenticatedRequest(nil, "POST", fmt.Sprintf("/api/latest/fleet/hosts/%d/lock", hostID), nil); err != nil {
+		return fmt.Errorf("lock host request: %w", err)
+	}
+	return nil
+}
+
+func (c *Client) MDMUnlockHost(hostID uint) error {
+	if err := c.authenticatedRequest(nil, "POST", fmt.Sprintf("/api/latest/fleet/hosts/%d/unlock", hostID), nil); err != nil {
```

Comment:
same as above, we need to send a response instead of `nil`, and probably change the method signature to return `unlockHostResponse.UnlockPIN`? I can do that second part as part of my work, no problem.

Reply:
I went ahead and added the PIN part!